src/main/java/graphql/execution2/Common.java

```diff
@@ -0,0 +1,72 @@
+package graphql.execution2;
```

Comment:
Common seems like a kitchen sink class.  Is there not another home or a  better named class maybe.